Stellar-ledger.x

```diff
@@ -484,11 +484,22 @@ struct LedgerCloseMetaV0
     SCPHistoryEntry scpInfo<>;
 };
 
+struct LedgerCloseMetaExtV1
+{
```

Comment:
There should be an `ExtensionPoint` here to allow for future extensions.

Reply:
I'm not sure whether this is the right way to go - wouldn't v2 extension be better? Working with nested extensions is not fun (from my recent experience of populating account ext v3 in Rust).

Reply:
We mostly use nested extensions at the moment, so I was thinking it'd make sense for consistency. It also lets you adapt by only adding new logic to handle the nested extension because they parent struct stays the same.

Reply:
There's only one field here though so a v2 extension is fine if that's the way you think we should go.

Reply:
Ok, I've added the extension point, we can decide later whether to use it.